region/maridia/outer.json

```diff
@@ -820,6 +820,10 @@
                     "h_canUsePowerBombs",
                     "canSuitlessMaridia",
                     "canTrickyJump"
+                  ],
+                  "note": [
+                    "Jump as the first action when breaking the tube to gravity jump to the top of the room.",
+                    "This can only be attempted once."
```

Comment:
```suggestion
                    "Jump as the first action after breaking the tube to gravity jump to the top of the room.",
                    "Open the door and go through it during the ascent. This can only be attempted once."
```